fixtures/eurolite/led-pix-12-hcl.json

```
@@ -397,127 +397,152 @@
         {
           "dmxRange": [10, 19],
           "type": "ColorPreset",
-          "comment": "Red"
+          "comment": "Red",
+          "colors": ["#ff0000"]
         },
         {
           "dmxRange": [20, 29],
           "type": "ColorPreset",
-          "comment": "Green"
+          "comment": "Green",
+          "colors": ["#00ff00"]
         },
         {
           "dmxRange": [30, 39],
           "type": "ColorPreset",
-          "comment": "Blue"
+          "comment": "Blue",
+          "colors": ["#0000ff"]
         },
         {
           "dmxRange": [40, 49],
           "type": "ColorPreset",
-          "comment": "White"
+          "comment": "White",
+          "colors": ["#ffffff"]
         },
         {
           "dmxRange": [50, 59],
           "type": "ColorPreset",
-          "comment": "Amber"
+          "comment": "Amber",
+          "colors": ["#ffbf00"]
         },
         {
           "dmxRange": [60, 69],
           "type": "ColorPreset",
-          "comment": "UV"
+          "comment": "UV",
+          "colors": ["#3e0694"]
         },
         {
           "dmxRange": [70, 79],
           "type": "ColorPreset",
-          "comment": "Red + Green"
+          "comment": "Red + Green aka Yellow",
+          "colors": ["#ffff00"]
         },
         {
           "dmxRange": [80, 89],
           "type": "ColorPreset",
-          "comment": "Red + Blue"
+          "comment": "Red + Blue aka Magenta",
+          "colors": ["#ff00ff"]
         },
         {
           "dmxRange": [90, 99],
           "type": "ColorPreset",
-          "comment": "Red + White"
+          "comment": "Red + White",
+          "colors": ["#ff8080"]
         },
         {
           "dmxRange": [100, 109],
           "type": "ColorPreset",
-          "comment": "Red + Amber"
+          "comment": "Red + Amber",
+          "colors": ["#ff6000"]
         },
         {
           "dmxRange": [110, 119],
           "type": "ColorPreset",
-          "comment": "Red + UV"
+          "comment": "Red + UV",
+          "colors": ["#9f034a"]
```

UV should be a separate color, see https://github.com/OpenLightingProject/open-fixture-library/blob/master/docs/capability-types.md#property-colors

```suggestion
          "colors": ["#9f034a", "#8800ff"]
```

(same for other capabilities)